pkg/shell.go

```diff
@@ -20,3 +23,82 @@ func ExecShellReturnStrings(command string, args ...string) (string, string, err
 	log.Println("Commad Execution STDERR: %s", errb.String())
 	return outb.String(), errb.String(), err
 }
+
+func ExecShellReturnStringsWithVars(osvars map[string]string, command string, args ...string) (string, string, error) {
+	
+	log.Printf("INFO: Running %s",command)
+	for k, v := range osvars {		
```

Comment:
I guess the shell calls are a temporary solution, what do you think to make it explicit the environment variables we're exposing?